<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JeuMainWindow</class>
 <widget class="QMainWindow" name="JeuMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>788</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>طاولة اللعب</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-image: url(:/Downloads/table.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>1011</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0" columnstretch="1,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>90</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>50</number>
     </property>
     <item row="2" column="3">
      <widget class="QLabel" name="card5">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="card3Robot">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="card4">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="card1Robot">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="cardMilieu">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="card2Robot">
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="card5Robot">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="card2">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="card6Robot">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="card7">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="typeCard">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="card3">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="card1">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="card4Robot">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="card7Robot">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="carteLabel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="card6">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
